environments/dev-env/README.md

```diff
@@ -0,0 +1,165 @@
+## Quickstart
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+For Mac OS:
+```
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+## 
+1. Clone this repo:
+
+```
+git clone -b dev-env https://github.com/dumb0002/edge-mc.git
```

Comment:
Should I clone the dev-env branch or the dev-env-v2 branch (being merged here)?

Reply:
Why do users clone a branch instead of `main`?

Reply:
@waltforme, just was a typo - fixed it
@MikeSpreitzer, I will update the readme for `main` branch after this PR is merged